cdk-test-project/README.md

```diff
@@ -0,0 +1,15 @@
+# Welcome to your CDK TypeScript project
```

Comment:
I'm wondering if we should include CDK as part of this effort, since it's not in scope, although it kind of comes "for free" with CloudFormation support. It might require more extensive testing with different CDK projects, but would also require documentation, etc., so maybe we can make it a separate feature and PR!

Reply:
Sure, removed CDK.